package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "communication-services-web-chat-hero",
+  "version": "1.0.0",
+  "description": "--- page_type: sample languages: - javascript - nodejs - products: - azure - azure-communication-services ---",
```

Comment:
This is a very odd description.. can we update to something more human friendly?
There is a [keywords](https://docs.npmjs.com/cli/v6/configuring-npm/package-json#keywords) field that might be more appropriate

Reply:
This is porting from what we had before. I think we can update the descriptions for both the calling and chat hero samples at a later time to maintain consistency.